### PR TITLE
Fix to make GIS scripts work in production environment

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**

### CODE REVIEW

File: src/pds/registry/utils/geostac/create_lola_pds4.py [hunk 1]
```diff
@@ -82,15 +83,16 @@ def create_product_external(item):
...
     with importlib.resources.open_text(templates, "product-external-template.xml") as io:
         template_text = io.read()
         template = env.from_string(template_text)
 
         item_title = item["assets"]["data"]["title"]
 
         last_slash_i = item["assets"]["data"]["href"].rfind("/")
-        file = item["assets"]["data"]["href"][last_slash_i + 1:]
+        file = "data/" +item["assets"]["data"]["href"][last_slash_i + 1:]
+        print(f'file is on {item["assets"]["data"]["href"]},fake file is on {file}')
```
Comment (nutjob4life) [inline src/pds/registry/utils/geostac/create_lola_pds4.py:94]: Should this be using `logging.info` instead? If not, should it be using `print(…, file=sys.stderr)` instead?

File: src/pds/registry/utils/geostac/create_lola_pds4.py [hunk 2]
```diff
@@ -147,7 +149,10 @@ def create_product_browse(item):
         item_title = item["assets"]["data"]["title"]
 
         last_slash_i = item["assets"]["thumbnail"]["href"].rfind("/")
-        file = item["assets"]["thumbnail"]["href"][last_slash_i + 1:]
+        file = "data/" + item["assets"]["thumbnail"]["href"][last_slash_i + 1:]
+        print(f'file is on {item["assets"]["thumbnail"]["href"]},fake file is on {file}')
```
Comment (nutjob4life) [inline src/pds/registry/utils/geostac/create_lola_pds4.py:153]: Same as above

**Review by nutjob4life — APPROVED** [review]
Approved, but do see my `logging` comment 😁

File: src/pds/registry/utils/geostac/create_lola_pds4.py (hunk 2 above)
Comment (jordanpadams) [reply src/pds/registry/utils/geostac/create_lola_pds4.py]: Updated

**Review by nutjob4life — APPROVED** [review]

**Comment by jordanpadams** [comment]
Tests are failing, but will create a new ticket if that migrates along to unstable build

**Pull request merged**